sklearn/model_selection/tests/test_validation.py

```diff
@@ -2272,9 +2279,15 @@ def test_cross_validate_failing_scorer(
             )
             for key in results:
                 if "_score" in key:
-                    # check the test (and optionally train score) for all
-                    # scorers that should be assigned to `error_score`.
-                    assert_allclose(results[key], error_score)
+                    if "2" in key:
```

Comment:
I see that we already check `_score`, but I think using `score_2` is more explicit:

```suggestion
                    if "score_2" in key:
```